hermes/probe/read/read_file_test.go

```diff
@@ -0,0 +1,80 @@
+package read
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/create"
+	"github.com/googleinterns/step224-2020/hermes/probe/fakegcs"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+
+	metricpb "github.com/google/cloudprober/metrics/proto"
+	probepb "github.com/googleinterns/step224-2020/config/proto"
+	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+func TestReadFile(t *testing.T) {
+	ctx := context.Background()
+	bucketName := "test_bucket_probe0"
+	client := fakegcs.NewClient()
+	fbh := client.Bucket(bucketName)                         // fakeBucketHandle
```

Comment:
Make the name more descriptive, and delete the comment.

Reply:
+1
`bucketHandle` would do.

Reply:
changed to fakeBucketHandle comment deleted